### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alanyee**
Made explicit HTTPS calls and change absolute link to relative link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ You can ask questions and join the development discussion:
 - On the [Keras Google group](https://groups.google.com/forum/#!forum/keras-users).
 - On the [Keras Slack channel](https://kerasteam.slack.com). Use [this link](https://keras-slack-autojoin.herokuapp.com/) to request an invitation to the channel.
 
-You can also post **bug reports and feature requests** (only) in [Github issues](https://github.com/fchollet/keras/issues). Make sure to read [our guidelines](https://github.com/fchollet/keras/blob/master/CONTRIBUTING.md) first.
+You can also post **bug reports and feature requests** (only) in [Github issues](https://github.com/fchollet/keras/issues). Make sure to read [our guidelines](CONTRIBUTING.md) first.
```
Comment (fchollet) [inline README.md]: URLs should be explicit, as this code is used in more than one place.

Comment (alanyee) [reply README.md]: If they pull the README.md, shouldn't they also have pulled the CONTRIBUTING.md?

Comment (fchollet) [reply README.md]: No.

File: README.md [hunk 2]
```diff
@@ -184,3 +184,7 @@ Keras was initially developed as part of the research effort of project ONEIROS
 >_"Oneiroi are beyond our unravelling --who can be sure what tale they tell? Not all that men look for comes to pass. Two gates there are that give passage to fleeting Oneiroi; one is made of horn, one of ivory. The Oneiroi that pass through sawn ivory are deceitful, bearing a message that will not be fulfilled; those that come out through polished horn have truth behind them, to be accomplished for men who see them."_ Homer, Odyssey 19. 562 ff (Shewring translation).
 
 ------------------
+
+## License
+
+Read the [license](https://github.com/fchollet/keras/blob/master/LICENSE) for more information
```
Comment (fchollet) [inline README.md]: This part isn't necessary, the license is already mentioned prominently in the readme.

**Pull request merged**